sylvia-derive/src/lib.rs

```diff
@@ -621,6 +622,13 @@ fn interface_impl(_attr: TokenStream2, item: TokenStream2) -> TokenStream2 {
 /// and only for message types variants that resolves in an enum field,
 /// i.e. `exec`, `query` and `sudo`.
 ///
+/// ### `sv::features(...)`
+///
+/// Enables additional features for the contract. Allows user to use features that
+/// are considered breaking before the major release.
+///
+/// Currently supported features are:
+///     * `replies` - enables better dispatching of `reply` as well as its auto deserialization.
```

Comment:
That's a very humble description of a really useful and rich feature. I think of adding some URL to the docs at least, or ref to the rustdoc that broader explains replies.

Reply:
The docs are not yet there. I will try to expand the description though.